distinguished_contributors.md

```diff
@@ -0,0 +1,16 @@
+# Jupyter Distinguished Contributors
+## Purpose
+This document describes the Distinguished Contributors body of the Jupyter Governance Model. Membership in the Distinguished Contributors is meant to recognize the work of community members that have gone above-and-beyond in their work on the project. It is not meant as a means of conveying power or responsibility. A public record of each Distinguished Contributor’s current biography and achievements will be maintained by the members themselves and will be displayed on the Jupyter website.
+## Rights of Distinguished Contributors
+Distinguished Contributors have the right to elect new Distinguished Contributors. They also have the right to participate in other occasional votes specified by the broader Jupyter Governance Model.
```

Comment:
@afshin This is specifically where I'm finding Distinguished Contributors unclear. You mentioned in the response that it's not a governance body but as written this is a bit confusing.

Reply:
That's a really good point!

This was a late addition and the idea behind it was that occasionally the Board of Directors and the Software Steering Council might jointly want to subject something to a project-wide vote. We didn't know what that might be and we didn't have a specific example in mind (although maybe @fperez did, I believe he raised the idea originally). But this sentence was meant to indicate that the Distinguished Contributors are not really part of the governance though they may on occasion be solicited for a vote.

Reply:
I would suggest removing as the governing body would always have the right to ask Distinguished Contributors to vote on any governance issues that the governing body requests.

Reply:
Leaving the sentence about electing new Distinguished Contributors.

Reply:
I agree with you. I'd like to ask Fernando what scenario he was considering in tomorrow's call, but I think removing it would be clarifying.

Reply:
I think part of this is also related to managing the expectations of the people that become Distinguished Contributors. Is being a Distinguished Contributor simply an award of recognition that they receive or are they expected to participate and do something once they have achieved that recognition? Having access to this group of people could be a good way to get feedback that could inform the Board of Directors and Software Steering Council so it would be good to indicate this expectation somewhere so it is clear what could be asked of Distinguished Contributors.

Reply:
Good point @sharanf. Perhaps it is better to move away from governance terms and have something more along the lines:

We recognize the knowledge and experience that the Distinguished Contributor brings to the project. From time to time, the Board of Directors and Software Steering Council may poll the group of Distinguished Contributors for their insights.

Reply:
My concern is if you leave it in governance terms,  it becomes a governing body. Therefore, adding to the confusion when rolling out the new governance.